.env.sample

```diff
@@ -1,3 +1,4 @@
 REACT_APP_CONTENTSTACK_API_KEY=YOUR_STACK_API_KEY
 REACT_APP_CONTENTSTACK_DELIVERY_TOKEN=YOUR_DELIVERY_TOKEN
-REACT_APP_CONTENTSTACK_ENVIRONMENT=YOUR_ENVIRONMENT
\ No newline at end of file
+REACT_APP_CONTENTSTACK_ENVIRONMENT=YOUR_ENVIRONMENT
+REACT_APP_CONTENTSTACK_REGION=YOUR_STACK_REGION
```

Comment:
Let's add available region values in this file also as a comment above REACT_APP_CONTENTSTACK_REGION

Reply:
ok

Reply:
@praveen-mohan-cs , can you review now